npm/react/cypress/component/basic/rerender/effects.cy.jsx

```diff
@@ -62,7 +62,12 @@ it('should run unmount effect cleanup when unmounting', () => {
 
     cy
     .then(() => ReactDom.unmountComponentAtNode(getContainerEl()))
-    .then(() => {
+    .then(async () => {
+      // does not call useEffect in react 17 unmount synchronously.
+      // @see https://github.com/facebook/react/issues/20263.
```

Comment:
Period made the link not clickable
```suggestion
      // @see https://github.com/facebook/react/issues/20263
```